### PR TITLE
chore: use require for cookie tests

### PR DESCRIPTION
**Pull request by AtofStryker**
- Closes N/A

### User facing changelog
N/A
### Additional details

Low hanging fruit to move `makeRequest` in the `cookie_behavior` tests to utils and leverage `Cypress.require` to a) have a single source of truth and b) start to leverage `Cypress.require` within some of our tests

### PR Tasks

- [x] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Review by chrisbreiding — APPROVED** [review]

File: packages/driver/cypress/e2e/e2e/origin/cookie_behavior.cy.ts [hunk 1]
```diff
@@ -1,34 +1,6 @@
...
-      xhr.onerror = function () {
-        reject(xhr.response)
-      }
-
-      xhr.send()
-    })
-  }
+import { makeRequestForCookieBehaviorTests as makeRequest } from '../../../support/utils'
 
+describe('Cookie Behavior with experimentalSessionAndOrigin=true', () => {
```
Comment (emilyrohrbough) [inline packages/driver/cypress/e2e/e2e/origin/cookie_behavior.cy.ts]: with experimentalSessionAndOrigin=true is implied by being in origin folder
```suggestion
describe('Cookie Behavior', () => {
```

File: packages/driver/cypress/support/utils.js [hunk 2]
```diff
@@ -137,6 +137,36 @@ export const expectCaret = (start, end) => {
   }
 }
 
+export const makeRequestForCookieBehaviorTests = (
```
Comment (emilyrohrbough) [inline packages/driver/cypress/support/utils.js:144]: We lost the types associated with this. Can we update utils.js to utils.ts and/or add this as its own typed' file?

File: packages/driver/cypress/e2e/e2e/origin/cookie_behavior.cy.ts [hunk 3]
```diff
@@ -59,35 +31,7 @@ describe('Cookie Behavior with experimentalSessionAndOrigin=true', () => {
...
-                  reject(xhr.response)
-                }
-
-                xhr.send()
-              })
-            }
+            const { makeRequestForCookieBehaviorTests: makeRequest } = Cypress.require('../../../support/utils')
 
             // @ts-ignore
             window.makeRequest = makeRequest
```
Comment (emilyrohrbough) [inline packages/driver/cypress/e2e/e2e/origin/cookie_behavior.cy.ts:37]: Why attach this to window? Can't the tests using this just use Cypress.require directly?

File: packages/driver/cypress/support/utils.js (hunk 2 above)
Comment (AtofStryker) [reply packages/driver/cypress/support/utils.js:144]: I was kind of bummed about this. Changing the extension on utils.ts might be a bit of a haul, so I wonder if we just want to create a typed file in `support` with this function in it to preserve the types?

File: packages/driver/cypress/e2e/e2e/origin/cookie_behavior.cy.ts (hunk 3 above)
Comment (AtofStryker) [reply packages/driver/cypress/e2e/e2e/origin/cookie_behavior.cy.ts:37]: They can. This is just for convenience so I don't have to redeclare the require in each single `origin` block.

File: packages/driver/cypress/e2e/e2e/origin/cookie_behavior.cy.ts (hunk 1 above)
Comment (AtofStryker) [reply packages/driver/cypress/e2e/e2e/origin/cookie_behavior.cy.ts]: I feel like we can drop the flag, but I wonder if something more than `Cookie Behavior` is needed to capture what the spec is actually doing. Then again simpler might be better?

File: packages/driver/cypress/support/utils.js (hunk 2 above)
Comment (emilyrohrbough) [reply packages/driver/cypress/support/utils.js:144]: I just renamed it it locally, and doesn't seem to be having massive issues. But then my VSCode extension could be slow. Seems to only partially work in our setup.

Comment (AtofStryker) [reply packages/driver/cypress/support/utils.js:144]: If it isn't very large, I can give it a shot. Main reason I was avoiding it was blowing the scope of this PR up, but if it is fairly simple I don't see why not!

**Review by mjhenkes — APPROVED** [review]

**Comment by AtofStryker** [comment]
@emilyrohrbough I finally got this up to date with `require`. This keeps the types and just converts the `utils` file over to typescript

**Review by chrisbreiding — APPROVED** [review]

**Pull request merged**